wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -696,132 +699,94 @@ body:not(.title-tagline-hidden) .site-branding-text {
+	display: inline;
 }
 
-@media screen and (max-width: 48em) {
-	.wp20-locale-switcher-container:hover {
-		background: var(--wp20--color--active-menu);
-	}
+.wp20-locale-switcher-container select {
+	visibility: hidden; /* Prevent FOUC and iOS native picker UI */
 }
 
-@media screen and (max-width:48em) {
-	.wp20-locale-switcher-container.is-toggled {
-		background: var(--wp20--color--active-menu);
+@media screen and (min-width: 769px) {
+	.autocomplete__wrapper {
+		display: flex;
+		flex-direction: column;
+		align-items: flex-end;
+		z-index: 1;
 	}
-}
 
-.wp20-locale-switcher-container label {
-	display: inline;
-}
-
-.wp20-locale-switcher-container select {
-	visibility: hidden; /* Prevent iOS native picker UI */
+	.autocomplete__wrapper > * {
+		max-width: 250px;
+	}
 }
 
 @media screen and (max-width: 48em) {
-	.wp20-locale-switcher-container .select2-container {
-		width: 100% !important; /* override selectWoo rule. locale-swithcer.js L24. */
+	.wp20-locale-switcher-container:hover {
+		background: var(--wp20--color--active-menu);
 	}
 }
 
-.wp20-locale-switcher-container .select2-container--default .select2-selection--single {
+input.autocomplete__input,
+.autocomplete__hint {
+	display: block;
 	border: none;
-	height: 60px;
+	text-align: end;
+	padding: var(--wp20--spacing--menu-spacing) 36px var(--wp20--spacing--menu-spacing) var(--wp20--spacing--xsmall);
+	font-size: 14px;
+	line-height: 26px;
+	border-radius: 0;
 	background: transparent;
-	text-align:right;
-	display: flex;
-	align-items: center;
-	justify-content: flex-end;
 }
 
-/* The select language label */
-.wp20-locale-switcher-container .select2-selection .select2-selection__rendered {
-	font-size: 14px;
-	max-width: 150px;
-	text-overflow: ellipsis;
+input.autocomplete__input::placeholder {
+	color: var(--wp20--color--text-meta);
 }
 
-/* Remove the ellipsis overflow */
-@media screen and (min-width: 375px) {
-	.wp20-locale-switcher-container .select2-selection .select2-selection__rendered {
-		max-width: unset;
-	}
+.autocomplete__input.autocomplete__input--focused,
+.autocomplete__input.autocomplete__input--focused:focus {
+	background-color: white;
+	box-shadow: none;
+	outline: none;
 }
 
-.wp20-locale-switcher-container .select2-container .select2-selection--single .select2-selection__rendered {
-	padding-right: 22px !important;
-	color: var(--wp20--color--grey-6);
+/* When both the hint and input are displayed stacked */
+span + .autocomplete__input.autocomplete__input--focused {
+	border: 1px solid var(--wp20--color--border);
+	border-bottom: none;
+	text-align: start;
+	background-image: url('images/search-icon.svg');
+	background-repeat: no-repeat;
+	background-size: 13px;
+	background-position: calc(100% - 20px) center;
+	padding-top: 13px;
+	padding-bottom: 13px;
+	line-height: 1.4;
 }
 
-.wp20-locale-switcher-container .select2-container--open .select2-selection--single .select2-selection__rendered {
+.autocomplete__hint {
```

Comment:
I'm not sure why, but the first word isn't capitalized in the hint, even though it is in the raw data and in the dropdown:

<img width="455" alt="Screenshot 2023-05-03 at 8 25 19 AM" src="https://user-images.githubusercontent.com/484068/235963079-4dd248ec-8b1d-45c7-9ed9-ec24e3e00a93.png">

It's odd because the text doesn't seem to show up in the DOM, and it's not set via CSS `content` either 🤔 I checked the Autocomplete source and it wasn't obvious there.

I don't think it's related to any of the styles, and it was happening in #71 too.

`text-transform: capitalize` works in some cases, but would be wrong in ones like `Español De Colombia` where the `De` should be lowercase.

We are using the hint in an unintended way in order to match the design, but I'm not sure why that'd be related 🤷🏻‍♂️

Reply:
I only get lowercase when I've entered lowercase in the input.

Looks like it might be this line, where the input text replaces the start of the option https://github.com/alphagov/accessible-autocomplete/blob/main/src/autocomplete.js#L452

`hintValue` is bound here https://github.com/alphagov/accessible-autocomplete/blob/main/src/autocomplete.js#L490

Reply:
Ah, good find. Yeah, `jQuery( '.autocomplete__hint' ).val()` shows the value, but FF/Chrome inspectors don't 🤷🏻‍♂️ 

Doesn't look like anything we can change easily, so I'd say it's just a quirk, and not worry about it 👍🏻 

If we really have to I guess we could set up a mutation observer to overwrite it w/ the correct capitalization, but that may have a11y consequences, etc. I'm sure there's some reason they're doing the `substr` bit. I guess if we have to do that, it'd be better to just add our own custom element for sighted users, and leave the hint hidden for screen readers. The observer could copy the hint value into the custom element.